### DISCUSSION TITLE
How to change lotus id

### DISCUSSION THREAD

**Discussion by yzn5555** [filecoin-project/lotus#7823] [General Q&A]
Now the two lotus nodes query the net id to be the same。The reason should be that the data of one of the nodes is completely copied from the other node. Now that the IDs of the two nodes are the same, will there be a problem?
There is another question. Can I manually add a peer-to-peer connection. The peer is a node. It is good to get the network ID of the other party, and then can I synchronize the node data through the other node after adding it?

**Comment by Reiers** [comment]
Hi @yzn5555 

Thanks for the report.
Moving this issue to Lotus Discussions for help and troubleshooting.

Closing Lotus issue ticket.

**Comment by LexLuthr** [comment]
@yzn5555 AFAIK, I don't think 2 nodes can have the same ID at the same time in the network.
About your second question, can you clarify a bit more as to what are you looking to do with the peer? Also, why do you want to run 2 nodes with same ID at the same time?

**Reply by LexLuthr** [reply]
I am archiving this discussion as there has been no new input from the author for more than 7 days.
